power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/SendMoneyImpl.java

```diff
@@ -0,0 +1,10 @@
+package com.adorsys.gis.powerpay.powerpaybackend.services;
+
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+@Component
+@Service
```

Comment:
Same

Reply:
ok sir noted

Reply:
hello sir @stephane-segning changes updated requesting for review